homeassistant/components/quantum_gateway/__init__.py

```diff
@@ -1 +1,56 @@
 """The quantum_gateway component."""
+
+from requests import RequestException
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import DOMAIN as DEVICE_TRACKER_DOMAIN
+from homeassistant.const import CONF_HOST, CONF_PLATFORM
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DATA_COODINATOR, DOMAIN, LOGGER, PLATFORM_SCHEMA
+from .coordinator import QuantumGatewayCoordinator
+
+CONFIG_SCHEMA = vol.Schema(
+    {DEVICE_TRACKER_DOMAIN: [PLATFORM_SCHEMA]},
+    extra=vol.ALLOW_EXTRA,
+)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Quantum Gateway component."""
```

Comment:
We should maintain the current entrypoint, so we should use the device_tracker async_setup_platform instead

Reply:
Hey Joost, I'm not sure what you mean. Should be initializing the coordinator in the device_scanner async_get_scanner by reverting this change?
https://github.com/home-assistant/core/pull/146007/commits/ff81336580fbe7f32cb3a58188ddbf5c621fff1c#diff-2b177940cc8b47dd3c46b82c10defa53607dd9898e1cdfa4b89291c1edc8b9eaL30-L43

My reasoning for moving the initialization here was to reduce the amount of changes when switching to a config entry, but I'm happy to switch it back if this is what you mean.